assignment/scripts/aboutMe.js

```diff
@@ -47,12 +64,30 @@
 // be sure to think through all the scenarios. 
 // console.log `mostPets` after the conditional has run.
 
+let mostPets='Most Pets';
+if(pets>=friendsPets){
+    console.log('I have the', mostPets)
+}
+else if(friendsPets>=pets){
+    console.log('My friend has the ', mostPets)
+}
 // 17 - Write a *switch* statement that logs:
 //      "First is the worst" if your lucky number is 1
 //      "Second is the best" if your lucky number is 2
 //      "Third is the one with the polka dot dress" if your lucky number is 3
 //      Otherwise, log "Luck is what happens when preparation meets opportunity"
 //      You'll need to research how to use switch statements!
+if(luckyNumber===1){
```

Comment:
Careful! Question 17 specified a switch statement. You used an if/else. It has the same effect, but does not satisfy the requirement.